custom-vu/src/main/kotlin/jces1209/vu/page/AttachScreenShot.kt

```diff
@@ -0,0 +1,26 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.*
+import org.openqa.selenium.interactions.Actions
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+abstract class AttachScreenShot(
+    protected val driver: WebDriver
+) {
+
+    fun makeScreenShot() {
+        val src = (driver as TakesScreenshot).getScreenshotAs(OutputType.BYTES)
+    }
+
+    fun pasteScreenShot() {
+        val builder = Actions(driver)
+        builder.keyDown(Keys.CONTROL).sendKeys("v").perform()
+    }
+
+    fun waitForTheScreenShotAttached() {
+        driver.wait(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//*[@data-test-id=" +
```

Comment:
The issue could already have some attachments
It's necessary to verify the number of attachments before and after

also, the attachment could be created but not loaded - it's necessary to wait for `data-test-status="complete"` for the element with `data-testid="media-file-card-view"`
this is for Cloud

Reply:
It seems this was addressed, but in the parent class, so for both Cloud and DC, while the feedback was for Cloud only.
It's fine to abandon inheritance entirely and just have independent implementations for Cloud and DC. The only common part seems to be `makeScreenShot`, which could be removed from the interface entirely.